student.rb

```diff
@@ -0,0 +1,12 @@
+class Student < Person
+  attr_reader :classroom
+
+  def initialize(classroom)
+    super(18, 'john', parent_permission: true)
+    @classroom = classroom
+  end
```

Comment:
- [ ] with the previous comment I meant that the parameters need to be added to the arguments the initialize method will accept, this will make it look something like below:
```
  def initialize(age, classroom, name = "Unknown", parent_permission = true)
    super(age, name, parent_permission)
    @classroom = classroom
  end
```